evm/bridge-token-factory/contracts/test/TestWormhole.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.24;
+
+contract TestWormhole {
```

Comment:
https://github.com/wormhole-foundation/wormhole/blob/a3284d66d5da7c0480262d5f7c93d7cc8322f6a1/ethereum/contracts/Implementation.sol#L15

Maybe use an implementation closer to the real one. It seems to be fairly simple.

Reply:
I'd rather keep it simple. We just want to test that we pass correct parameters and we can do it with this simple setup